GameFiles/IW4x/userraw/scripts/_integration.gsc

```diff
@@ -653,7 +668,37 @@ TeamSwitchImpl()
     return self.name + "^7 switched to " + self.team;
 }
 
-HideImpl()
+LockControlsImpl( target )
+{
+    if ( !IsAlive( self ) )
+    {
+        return self.name + "^7 is not alive";
+    }
```

Comment:
This was incorrect as it would just return your name, but making the above proposed changes would fix this.